liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/CleanServerMojo.java

```diff
@@ -24,31 +24,31 @@
  */
 @Mojo(name = "clean-server")
 public class CleanServerMojo extends StartDebugMojoSupport {
-    
+
     /**
      * Clean the logs directory.
      */
-    @Parameter(property = "cleanLogs", defaultValue = "true")
-    private boolean logs = true;
-    
+    @Parameter(alias = "logs", property = "cleanLogs", defaultValue = "true")
+    private boolean cleanLogs = true;
+
     /**
      * Clean the workarea directory.
      */
-    @Parameter(property = "cleanWorkarea", defaultValue = "true")
-    private boolean workarea = true;
-    
+    @Parameter(alias = "workarea", property = "cleanWorkarea", defaultValue = "true")
+    private boolean cleanWorkarea = true;
+
     /**
      * Clean the dropins directory.
      */
-    @Parameter(property = "cleanDropins", defaultValue = "false")
-    private boolean dropins = false;
-    
+    @Parameter(alias = "dropins", property = "cleanDropins", defaultValue = "false")
```

Comment:
So does the "alias" allow this to be configured as `<dropins>` in the pom.xml? Or simply allow them to specify the property value using -Ddropins? I want to make sure they can still use <dropins> in pom.xml. Otherwise this is a breaking change.

Reply:
It allows for the configuration as `dropins` in the pom.